src/resources/x509.cc

```diff
@@ -101,7 +101,9 @@ PRIMITIVE(parse) {
     String* str = String::cast(input);
     data = reinterpret_cast<const uint8_t*>(str->as_cstr());
     length = str->length() + 1;
-    ASSERT(data[length - 1] == '\0');
+    // Should not happen since Toit strings are null terminated.
```

Comment:
I think this should remain an ASSERT if it shouldn't happen and the VM is supposed to guarantee it.